Allow users to import the globalCSS themselves (#56)
withstudiocms/ui: docs/src/content/docs/docs/guides/customization.mdx

Below, you will find all the CSS variables used in StudioCMS UI. Always make sure to provide them as 
HSL values without commas. You can use them in your CSS like this: `background-color: hsl(var(--background-base));`.

## Disabling CSS Injection

If you want to disable the CSS injection, you can do so by setting the `noInjectCSS` option to `true`:

```ts twoslash showLinenumbers title="astro.config.mjs" {7}
import { defineConfig } from 'astro/config';
import ui from '@studiocms/ui';

export default defineConfig({
  integrations: [
    ui({
      noInjectCSS: true,
    })
  ]
});
```

> You can then include the CSS in your project manually. (Useful if you want to mix studiocms-ui with other CSS frameworks. or if you are using the UI styling only for a subset of your components.)

Reviewer: ```suggestion
You can then include the CSS in your project manually. This is useful if you want to mix StudioCMS UI with other CSS frameworks, or if you are using the UI styling only for a subset of your components.
```

```astro showLinenumbers title="src/layouts/SUILayout.astro"
---

> // Relying on the Virtual module to import the CSS file (recommended)

Reviewer: ```suggestion
// Using the virtual module to import the CSS file (recommended)
```

import "studiocms:ui/global-css";

> // Or import the CSS file directly from the package

Reviewer: ```suggestion
// Alternatively, import the CSS file directly from the package
```
